src/main/kotlin/org/neo4j/graphql/handler/filter/OptimizedFilterHandler.kt

```diff
@@ -9,13 +9,18 @@ import org.neo4j.graphql.handler.projection.ProjectionBase
 import org.neo4j.opencypherdsl.*
 import org.neo4j.opencypherdsl.Cypher
 import org.neo4j.opencypherdsl.Node
-import org.slf4j.LoggerFactory
+
+
+typealias WhereClauseFactory = (
```

Comment:
please add type aliases also for the other long names from the DSL, makes it much easier to read
or at least an static import